src/components/GetStarted/EditPreferences.vue

```diff
@@ -181,12 +181,67 @@ export default {
 			default: 0,
 		},
 	},
+	data() {
+		return {
+			causeList: [
+				{
+					id: 5,
+					name: 'women',
+					checked: false,
+				},
+				{
+					id: 6,
+					name: 'shelter',
+					checked: false,
+				},
+				{
+					id: 4,
+					name: 'education',
+					checked: false,
+				},
+				{
+					id: 102,
+					name: 'technology',
+					checked: false,
+				},
+				{
+					id: 8,
+					name: 'agriculture',
+					checked: false,
+				},
+				{
+					id: 96,
+					name: 'COVID-19',
+					checked: false,
+				},
+				{
+					id: 25,
+					name: 'health',
+					checked: false,
+				},
+				{
+					id: 32,
+					name: 'refugees',
+					checked: false,
+				},
+				{
+					id: 29,
+					name: 'arts',
+					checked: false
+				}
+			]
+		};
+	},
 	computed: {
 		causesTrimmed() {
-			if (this.causes.length <= 3) {
-				return this.causes;
+			const causeIds = this.causes.map(cause => cause.id);
```

Comment:
Alternatively, we can just pass the id's into this component from `GetResults` since that is all this component needs and we could remove this map.

Reply:
We need the shortened names from the map, I just enabled the id's so i could easily map from the LoanChannel based values (names are different here) to the causeList values which dictate the image paths.